lib/campaigns.js

```diff
@@ -4,6 +4,12 @@ var campaigns = {
     lang: 'de',
     newsletters: 'mozilla-foundation',
     source_url: 'https://petitions.mozilla.org/data-retention/'
+  },
+  'stay-secure': {
+    bsd: '',
```

Comment:
I think my only question is: if this is a petition, how are you storing firstname and lastname? I didn't think basket was capable of that, if it is, I am interested in how as I would use it elswhere :)